src/test/resources/Features/OCD-2102-SearchSurveillanceListingsWithChplIDVersion.feature

```diff
@@ -0,0 +1,17 @@
+@Regression
+Feature: OCD-2102 - Allow surveillance search for listings with CHPL ID version
+
+Background: User is logged in and navigates to Manage Surveillnace Activity section
+	Given I am on CHPL admin page	
+	When I fill in username
+	And I fill in Password
+	When I click Log in button
+	Then I navigate to Manage Surveillnace Activity section
```

Comment:
I don't like this as a "background" thing. Part of it might be that I don't like our current admin nav section, but even without that, the GWAWT should probably be something more like GWAAT.

Reply:
I added it as background in case there was a scenario at later point to validate other fields in results. That said, it may be another feature altogether. I'll remove background and add it to scenario.